frontend/packages/knative-plugin/src/actions/add-event-sink.tsx

```diff
@@ -31,7 +23,7 @@ export const AddEventSinkAction = (
   return {
     id: 'event-sink-add',
     label: i18next.t('knative-plugin~Event Sink'),
-    icon: <EventSourceIcon />,
+    icon: <SignInAltIcon />,
```

Comment:
Can we reuse `eventSourceIcon` from `../utils/icons.tsx` here?

This makes the code more readable and we have just one place where we define the icon.